metadata.json

```
@@ -1,6 +1,6 @@
 {
   "name": "puppet-icinga2",
-  "version": "5.0.2-rc0",
+  "version": "6.0.1-rc0",
```

```suggestion
  "version": "6.0.0",
```

This needs to stay at 6.0.0. The release rake task will bump it to 6.0.1-rc0

---

Ich bin verwirrt, hat der Rake Task prepare:release das getan?

---

I'm not sure which commands you executed in which order. If you want to prepare a release:
* bump version in metadata.json to desired version
* set CHANGELOG_GITHUB_TOKEN env variable with a github token
* run `bundle exec rake release:prepare`

That will generate the CHANGELOG.md and REFERENCE.md, but it won't touch the metadata.json